src/keypair.rs

```diff
@@ -62,6 +66,7 @@ impl ReadWrite for Keypair {
                 writer.write_all(&key.to_vec())?;
                 writer.write_all(&key.public_key.to_vec())?;
             }
+            _ => return Err(anyhow!("Unsupported key type for write.")),
```

Comment:
No need to drop key type support right? Or are you explicitly trying to avoid K256 keys being read?

Reply:
Then do that explicitly instead of wildcarding it all?

Reply:
Fair enough